test/e2e/utils.go

```diff
@@ -191,6 +191,29 @@ func getJaegerOperatorImages(kubeclient kubernetes.Interface, namespace string)
 	return imageNamesMap, nil
 }
 
+func getJaegerOperatorNamespace() string {
+	if !usingOLM {
+		return namespace
+	}
+
+	var jaegerOperatorNamespace string
```

Comment:
Seems redundant, as only used to return empty string - could just `return ""` at the end?